vaadin-dev-server/src/main/java/com/vaadin/base/devserver/DevModeHandlerImpl.java

```diff
@@ -742,11 +746,13 @@ private void reuseExistingPort(int port) {
         command.add(webpackConfig.getAbsolutePath());
         command.add("--port");
         command.add(String.valueOf(port));
+        command.add("--watch-options-stdin"); // Tell wds to stop even if
+                                              // watchDog fail
         command.add("--env");
         command.add("watchDogPort=" + watchDog.get().getWatchDogPort());
         command.addAll(Arrays.asList(config
                 .getStringProperty(SERVLET_PARAMETER_DEVMODE_WEBPACK_OPTIONS,
-                        "-d --inline=false")
+                        "--devtool=eval")
```

Comment:
```suggestion
                        "--devtool=eval-source-map")
```

"eval" alone does not allow to see original code in the developer tools when debugging, which makes debugging in the browser harder. In practice, with this option I was unable to navigate to the `my-view.ts` file in the debugger, because it was merged with other files into a single source listing. 

The prior `-d` flag was documented as:
```
  -d           shortcut for --debug --devtool eval-cheap-module-source-map
               --output-pathinfo                                       [boolean]
```

The closest option [from this list](https://v4.webpack.js.org/configuration/devtool/#devtool) is "cheap-module-eval-source-map". "eval-source-map" is a bit slower, but includes a more detailed mapping.